src/components/Card/Card.vue

```diff
@@ -53,6 +41,30 @@ export default {
         })
         return nums
       }
+    },
+    deleteCard () {
+      const locationID = event.target.closest('article').id
+      this.$parent.$emit('removeCard', locationID)
+    },
+    refreshLocation () {
+      const cityState = this.locale.split(', ')
+      const location = { city: cityState[0], state: cityState[1] }
+      this.$parent.$emit('refreshLocale', location)
+    },
+    determineAqiMessage () {
+      if (this.aqi <= 50) {
+        return '🟢 Good'
+      } else if (this.aqi >= 51 && this.aqi <= 100) {
+        return '🟡 Moderate'
+      } else if (this.aqi >= 101 && this.aqi <= 150) {
+        return '🟠 Unhealthy for Sensitive Groups'
+      } else if (this.aqi >= 151 && this.aqi <= 200) {
+        return '🔴 Unhealthy'
+      } else if (this.aqi >= 201 && this.aqi <= 300) {
+        return '🟣 Very Unhealthy'
+      } else if (this.aqi >= 301) {
+        return '💀 Hazardous'
+      }
```

Comment:
Thanks for moving these out of that wild div and into this method - much better!